internal/commands/build.go

```diff
@@ -144,7 +144,7 @@ func Build(logger logging.Logger, cfg config.Config, packClient PackClient) *cob
 func buildCommandFlags(cmd *cobra.Command, buildFlags *BuildFlags, cfg config.Config) {
 	cmd.Flags().StringVarP(&buildFlags.AppPath, "path", "p", "", "Path to app dir or zip-formatted file (defaults to current working directory)")
 	cmd.Flags().StringVarP(&buildFlags.Builder, "builder", "B", cfg.DefaultBuilder, "Builder image")
-	cmd.Flags().StringVarP(&buildFlags.Registry, "buildpack-registry", "R", cfg.DefaultRegistry, "Buildpack Registry URL")
+	cmd.Flags().StringVarP(&buildFlags.Registry, "buildpack-registry", "r", cfg.DefaultRegistry, "Buildpack Registry URL")
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&buildFlags.Registry, "buildpack-registry", "r", cfg.DefaultRegistry, "Buildpack registry by name")
```

Reply:
@supra08 FYI, `cfg.DefaultRegistry` has been deprecated.   This should be `cfg.DefaultRegistryName`.  This is also why some of your Unit Tests are failing.